DIPs/dip-3.md

```diff
@@ -0,0 +1,132 @@
+---
+dip: 3
+title: Introduce EIP-1559 to Reform the Network Fee Market
+authors: Darwinia Network
+discussions-to: https://github.com/orgs/darwinia-network/discussions/1163
+status: Final
+type: Economic
+created: 2023-12-14
+---
+
+
+# DIP-3
+## Abstract
+This DIP suggests implementing `EIP-1559` within the Darwinia Network to transform its fee structure.
+
+Under this proposal, all transaction fees would be burned, while tips would be awarded directly to block authors.
+This aligns with our future inflation strategy, motivates collators, and paves the way for practical applications of *RING* and *KTON* tokens.
+The result is a more predictable, sustainable, and user-centric ecosystem.
+
+Before Darwinia adopts the new inflation model, this DIP will make minor adjustments to the inflation mechanism to prevent the reissuance of burned assets.
+
+
+## Rationale
+### Reasons
+1. By adopting `EIP-1559`, which replaces the gas auction system with a base fee plus a priority fee,
+   users will benefit from more predictable transaction costs while also reducing network congestion during peak times.
+   This improvement serves both individual users and the network overall.
+
+### Future Objectives
+1. To align with our upcoming inflation model,
+   the burning mechanism of `EIP-1559` for part of the transaction fees is consistent with Darwinia's strategy to revamp its inflation system.
+   This approach aims to maintain long-term token value stability and promote sustainable development of the network.
+2. To better balance responsibilities and rewards within the relay chain-parachain structure, where collators play a more indirect role in security,
+   it's logical to adjust block production incentives to reflect their updated contribution to consensus maintenance and network protection.
+3. To enhance utility for *RING* and *KTON* tokens, introducing tangible use cases is essential.
```

Comment:
KTON is not the target token of this DIP. so, considering removing KTON from it could be a better option.

Reply:
Agree